news/2022-08-31-the-sound-design-of-lazer.md

```diff
@@ -78,6 +77,8 @@ Regrettably, it may have been a mistake to not showcase more of these sound effe
 
 ---
 
-How familiar are you with the sound of lazer? If you haven't tried it yet, go give osu!(lazer) a try and listen to all the different sounds, which you can do by downloading it from [its GitHub releases page](https://github.com/ppy/osu/releases). osu!(lazer) is actively being worked on by peppy and many other individuals who contribute their time and knowledge, so if you have any suggestions or want to help yourself, feel free to browse around on GitHub or take a closer look at development in the [osu!dev Discord server](https://discord.gg/ppy)!
+How familiar are you with the sound of osu!(lazer)? If you haven't tried it yet, go give osu!(lazer) a try, which you can do by downloading it from [its GitHub releases page](https://github.com/ppy/osu/releases). 
+
+osu!(lazer) is actively being worked on by peppy and many other individuals who contribute their time and knowledge, so if you have any suggestions or want to help yourself, feel free to browse around on GitHub or take a closer look at development in the [osu!dev Discord server](https://discord.gg/ppy)!
```

Comment:
not sure what "help yourself" means here

Reply:
maybe it'd be better to word it like "so if you have any suggestions or want to participate..."

Reply:
'help out yourself' is probably what you were aiming for here

Reply:
aha. best to reword since i could not understand that after multiple reads.